### PR TITLE
GitHub Octovillage description

### PR DESCRIPTION
**Pull request by githubteacher**
@stoe @r-daga Can you look at this once it's complete and give a 👍  comment or reaction

### CODE REVIEW

File: githubteacher-octovillage.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# GitHub Teacher - Octovillage
+This is the hometown of the @githubteacher. It contains some much needed documentation to explain the @githubteacher.
```
Comment (stoe) [inline githubteacher-octovillage.md:2]: Would be great to include your picture.

Otherwise it looks ✨ . :shipit: 

:octocat: :sparkling_heart:

**Comment by r-daga** [comment]
Please let us know something about octovillage

**Comment by stoe** [comment]
> Please let us know something about octovillage

Maybe

**Comment by githubteacher** [comment]
> Maybe

@stoe not quite but sometimes. 😆

**Comment by githubteacher** [comment]
@stoe @r-daga can you review this one last time? I think it's ready to go

**Comment by stoe** [comment]
Perfect.

:shipit:

**Comment by githubteacher** [comment]
@midimonkey try not to close other people's pull requests 😉. I'll merge this in just a moment =)

**Comment by midimonkey** [comment]
@githubteacher Ok, sorry.  I'm surprised that others have the ability - I would have thought only the person who owns the commit has the permission.

**Comment by githubteacher** [comment]
@midimonkey not a problem! if we were using organizations I could have more granular control. That's another reason we want to use organizations too. I'll bring it up after this break too ❤️ 💯

**Comment by midimonkey** [comment]
@githubteacher Gotcha!

**Comment by superdev567** [comment]
jas likes this

**Comment by superdev567** [comment]
pacman

**Comment by githubteacher** [comment]
This got good feedback, and i was told to :shipit:. So i'm going to shipit.

**Comment by balakondepudi** [comment]
Need to fix this bug, 
/play bell

**Pull request merged**